adapters/Line/LineAdapter/GADMediationAdapterLineUtils.m

```diff
@@ -81,10 +84,32 @@
                                      : kFADNeedChildDirectedTreatmentFalse;
   }
   [config setNeedChildDirectedTreatment:needChildDirectedTreatment];
-  [FADSettings registerConfig:config];
+
+  FADAdLoader *adLoader = [FADAdLoader adLoaderForConfig:config outError:&error];
+  if (error) {
+    GADMediationAdapterLineLog(@"There was an error while initializingFADAdLoader: %@",
+                               error.localizedDescription);
+    return GADMediationAdapterLineErrorWithFiveAdErrorCode(error.code);
+  }
+  gAdLoader = adLoader;
+
   return nil;
 }
 
+FADAdLoader *_Nullable GADMediationAdapterLineFADAdLoaderForRegisteredConfig(
+    NSError *_Nullable *_Nullable errorPtr) {
+  if (!FADSettings.isConfigRegistered || !gAdLoader) {
```

Comment:
nitpick: 

`!FADSettings.isConfigRegistered` test is not needed because its result is the same as the existence check of `gAdLoader`.

---

`!FADSettings.isConfigRegistered` の検査は `gAdLoader` の存在チェックがあれば十分であるため不要です

Reply:
Okay! I kept it before just in case but removed.